views/js/assets/strategies.js

```diff
@@ -54,6 +54,20 @@ define([
      */
     var strategies = {
 
+        externalReplaced : {
+            name : 'externalReplaced',
```

Comment:
We should find a better name, like that reflect the fact we are looking for absolute url  packed in the item. So any absolute URL will be selected. 
Also add a small comment saying this strategy expects the assetManager manager to be fed with `setData('assets')`